pkg/util/mount/mount_linux.go

```diff
@@ -102,6 +102,11 @@ func doMount(source string, target string, fstype string, options []string) erro
 		glog.Errorf("Mount failed: %v\nMounting arguments: %s %s %s %v\nOutput: %s\n",
 			err, source, target, fstype, options, string(output))
 	}
+	// TODO: This is a temporary work around for Issue #7972. Once the underlying issue has been resolved, this can be removed.
+	_, udeverr := exec.Command("udevadm", "trigger").CombinedOutput()
```

Comment:
I wonder whether it's better to restrict this to the device that is being mounted/unmounted (using e.g. the --sysname_match flag)?  Otherwise all devices are triggered, and I believe that this potentially borks/remounts those other devices too (e.g. other PD's on the same node)?

http://manpages.ubuntu.com/manpages/karmic/en/man8/udevadm.8.html

Also, don't you need a --action flag on the unmount below?  The default action is "add" which doesn't seem right for unmount?